src/EnergyPlus/HeatingCoils.cc

```diff
@@ -1966,6 +1966,8 @@ namespace HeatingCoils {
 				EffHS = HeatingCoil( CoilNum ).MSEfficiency( StageNumHS );
 
 				PartLoadRat = min( 1.0, SpeedRatio );
+				HeatingCoil( CoilNum ).RTF = 1.0;
+
```

Comment:
Heating coil RTF report variable was always 0 in this defect file since a PLF curve was not used. All other models report RTF the same as PLF, or uses 1 for speeds higher than 1 for MS coils, if PLF curve is missing.